src/azure-cli/azure/cli/command_modules/profile/_validators.py

```diff
@@ -12,20 +12,20 @@
 def validate_tenant(cmd, namespace):
     """
     Make sure tenant is a GUID. If domain name is provided, resolve to GUID.
-    https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc#fetch-the-openid-connect-metadata-document
+    https://learn.microsoft.com/en-us/entra/identity-platform/v2-protocols-oidc#find-your-apps-openid-configuration-document-uri
     """
     from azure.cli.core.util import is_guid
     if namespace.tenant is not None and not is_guid(namespace.tenant):
         import requests
         active_directory_endpoint = cmd.cli_ctx.cloud.endpoints.active_directory
-        url = '{}/{}/.well-known/openid-configuration'.format(active_directory_endpoint, namespace.tenant)
+        url = '{}/{}/v2.0/.well-known/openid-configuration'.format(active_directory_endpoint, namespace.tenant)
```

Comment:
If all the functionality of this `validate_tenant(...)` function is merely detect the existence of that "well known" url and then extract the tenant_id, I believe this change is technically unnecessary.

That being said, now that you noticed this issue and send out this PR, I agree that we can make this change.

Reply:
> If all the functionality of this `validate_tenant(...)` function is merely detect the existence of that "well known" url and then extract the tenant_id, I believe this change is technically unnecessary.

True, but I am only trying to align with MSAL and reduce the invocation on v1 endpoint. I don't want to get a warning from Microsoft Entra ID some day in the future that Azure CLI is still depending on v1 endpoint to do its work.